test/Libraries/RevitTestServices/RevitTestServices.csproj

```diff
@@ -112,9 +112,13 @@
     </Reference>
   </ItemGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
-  <PropertyGroup>
-    <PostBuildEvent>xcopy /Y /R $(ProjectDir)TestServices.dll.config $(TestOutputPath)</PostBuildEvent>
-  </PropertyGroup>
+  <Target Name="AfterBuild">
```

Comment:
I was not able to build the solution because of issues with such way of copying in visual studio, so I replaced this with better way :)